Dockerfile

```diff
@@ -15,11 +18,12 @@ RUN \
       xmlstarlet \
       uuid-runtime \
       unrar \
```

Comment:
Do you really need `unrar` and all of these programs?

Reply:
Hi @ipeacocks, actually I am not an expert of the Plex Core and features, so I can't answer you about that.. Maybe @gbooker can bring you more details about that. 

I just added `udev` because can be useful to access external storage like in my Raspberry project

Reply:
A bit offtop. Why do you use 64 bit OS for Raspberry Pi. Raspberry advices to use 32-bits Raspbian even on 64-bits Pi 4.

Reply:
I use Pi 4 (RAM 4GB) with lots of containers on it (maria-db, postgre-db, web servers, python server, plex...) All of these is consuming lots of RAM and even more when several people are connected simultaneously. I am also using Ubuntu Server for ARM (no GUI), not Raspbian.

I haven't made deep benchmarks, but I can tell there is a significant gain in global performance and responsiveness using 64-bit with such a config. 32-bit architectures have per-process limits when it comes to use large amount of RAM.

Hope that's answering your question 🙂

Reply:
@ipeacocks just like @maxlabuche I also 64bit. I believe in the future, this will be the only option, its just a matter of time. I'd like to make the switch as early as possible. Plus the performance gains :)

Reply:
> Do you really need `unrar` and all of these programs?

`unrar` is needed for some of the third-party plugins (subtitles IIRC) but maybe it's not needed anymore.  The rest of the above are definitively needed.

> I just added `udev` because can be useful to access external storage like in my Raspberry project

`udev` doesn't work as you would expect in containers.  I'm skeptical of this and missed it in the first pass through.

Reply:
OK @gbooker for `udev`, we can skip it if not needed, I have also figured out other solutions. So what is missing to merge this PR? Do you have any concerns?

Reply:
In my experimentation, `udev` didn't work inside the containers.  We used to have it there but its lack of functionality prompted us to remove it.

We are still looking at getting our CI to build ARM images but that may take longer than expected.

Reply:
Thanks @gbooker , do not hesitate to add new comments.